aten/src/ATen/core/ivalue_inl.h

```diff
@@ -325,7 +325,7 @@ struct C10_EXPORT ivalue::Future : c10::intrusive_ptr_target {
       std::rethrow_exception(eptr_);
     }
 
-    postWaitHook(value_);
+    postWaitHook(value_, non_blocking);
```

Comment:
curious, how does `non_blocking` work when some pending CUDA kernels fail? When will the user see the error in with existing PyTorch CUDA APIs?